verif/sim/cva6.py

```diff
@@ -775,7 +775,7 @@ def parse_args(cwd):
                       help="switch AXI agent mode: yes for Active, no for Passive")
   parser.add_argument("--gen_sv_seed", type=int, default=0,
                       help="Run test N times with random seed")
-  parser.add_argument("--sv_seed", type=str, default="1",
+  parser.add_argument("--sv_seed", type=str, default=str(random.getrandbits(31)),
```

Comment:
If the seed is random, the CI tests will be executed with different seeds. This means the CI will not be deterministic. This can be a problem when executing Dhrystone or Coremark tests for instance for which we monitor cycle number.
Moreover some CI tests can have random behavior in relation to the seed. Debug can be more difficult.

Reply:
You can fix you're sv_seed using cva6.py option --sv_seed = n , by default sv_seed is always 1 and this limit finding bugs and use 100% of the UVM advantage (randomization), for now in CI all tests are running with only one sv_seed.

But we should store uvm seed in a specific file and store them in the artifacts